services/server/src/handlers/user.ts

```diff
@@ -313,7 +314,17 @@ async function findUsers(req: Request, res: Response, next: NextFunction) {
 
   const query: IUser.FindUsersApiQuery = findUsersQuery.parse(req.query);
   const result = await users.find(query);
-  const response: IUser.FindUsersApiResponse = result;
+
+  const isOnlineMap = await cache.onlineStatus.isOnlineBatch(result.list.map(user => user.id));
+  const resList = result.list.map<PopulatedSelf>((user) => ({
+    ...user, 
+    online: !!isOnlineMap.get(user.id)
+  }));
```

Comment:
Because we will remove the `online` filed from the response, we no longer need this change.